content/en/synthetics/browser_tests/actions.md

```diff
@@ -164,6 +164,45 @@ You can run browser tests within other browser tests, up to two levels of nestin
 
 **Note**: If it does not make sense for you to run your subtest independently, you can pause it. It will continue to be called as part of your main test, but it will not be executed individually.
 
+## HTTP Requests
+
+You can run HTTP requests as part of your Browser tests. 
+
+{{< img src="synthetics/browser_tests/recorder_http_requests.png" alt="HTTP Request step"  style="width:70%;" >}}
+
+### Perform an HTTP request
+
+To define your HTTP request:
+
+1. Choose the **Method** and **URL** to query. Available methods are: `GET`, `POST`, `PATCH`, `PUT`, `HEAD`, `DELETE`, and `OPTIONS`.
+2. Optionally specify **Advanced Options**:
+     * Follow redirects: Toggle to have the monitored endpoint follow up to ten redirects.
+     * Allow insecure certificates: Toggle to have your HTTP test go on with connection even if there is an error when validating the certificate.
+     * Headers: Defined headers override the default browser headers. For example, set the User Agent in the header to [identify Datadog scripts][1].
```

Comment:
```suggestion
     * Headers: Defined headers override the default browser headers, for example: set the User Agent in the header to [identify Datadog scripts][1].
```

Reply:
Removed the ref to user agent as a user agent is set by default by our tests